datafusion/sql/src/unparser/expr.rs

```diff
@@ -1799,6 +1899,38 @@ mod tests {
                 IntervalStyle::PostgresVerbose,
                 r#"INTERVAL '1 YEARS 7 MONS 0 DAYS 0 HOURS 0 MINS 0.00 SECS'"#,
             ),
+            (
+                interval_month_day_nano_lit(
+                    "1 YEAR 1 MONTH 1 DAY 3 HOUR 10 MINUTE 20 SECOND",
+                ),
+                IntervalStyle::MySQL,
+                r#"INTERVAL 1 YEAR + INTERVAL 1 MONTH + INTERVAL 1 DAY + INTERVAL 3 HOUR + INTERVAL 10 MINUTE + INTERVAL 20 SECOND"#,
```

Comment:
when this flatten to a sql, does it add a () around? 

e.g. we want this `datetime - (Interval 1 day + interval 3 hour)`

Reply:
Confirm unparser adds () around when there are several statements, for example

```sql
and o_orderdate < date '1993-07-01' + interval '6' month - interval '5' day
``` 
is translated to 
```sql
AND (`orders`.`o_orderdate` < ((CAST('1993-07-01' AS DATE) + INTERVAL 6 MONTH) - INTERVAL 5 DAY)))
```

Note: you can't wrap only interval as it does not work in MySQL, for example:
```sql
SELECT (NOW() + INTERVAL 1 MONTH) # works
SELECT (NOW() + (INTERVAL 1 MONTH)) # does not 
```

Reply:
Great catch, fixed